### PR TITLE
Feature #10 display name

### PR DESCRIPTION
**Pull request by BirgitBoss**
https://github.com/admin-shell-io/questions-and-answers/issues/10

Update Question "How is the relation between idShort, shortName and preferredName for a submodel element and the related concept description?"

Update question considering the introduction of displayName in V3.0RC01 of part 1.

### CODE REVIEW

**Comment by aorzelskiGH** [comment]
The Task Force prefers to have two seperate answers: one for idShort and one for displayName.

File: README.md [hunk 1]
```diff
@@ -109,13 +109,58 @@ Serial number is a submodel element of “Identification” submodel referenced
 
 Map concepts to “admin-shell.io” namespace (see [custom identifier best practices](#id18)). Within the concept description itself use “sourceOfDefinition” or “isCaseOf” to link to “original” concept. (Answered: 2020-08-24)
 
-**[How is the relation between idShort, shortName and preferredName for a submodel element and the related concept description?](#id30)** <a id="id30"></a><!-- ID: 30 -->
+**[Are there any best practices how to choose the idShort of an element?](#id30a)** <a id="id30a"></a><!-- ID: 30 -->
+
+For choosing an idShort value there are the following recommendations (rules and order can be used for automatically creation of an idShort):
+
+- If an English shortname is available in the related concept description, the English shortName can be used as idShort. 
+- If there is an English preferredName available in the related concept description, this name can be converted to idShort, e.g. the preferredName “Maximum rotation speed” can be converted to “maximumRotationSpeed”.
```
Comment (BirgitBoss) [inline README.md:117]: " “maximumRotationSpeed” ==>  “MaximumRotationSpeed”.

**Review by BirgitBoss — COMMENTED** [review]
2021-05-03 Check whether old questions (displayname etc. and OPC UA DI) now removed? Otherwise the Exemplary Modeling WG agreed to the split of the original questions and the changes.

**Comment by BirgitBoss** [comment]
Changes as agreed in Exmplary Modelling Meeting incorporated: ready to be merged into main branch from my point of view.

**Pull request merged**